layer-publisher/src/layer-publisher-stack.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/explicit-member-accessibility */
```

Comment:
(Optional) A bit of background in a comment on why we need to access a private/protected(?) variable?